packages/documentation/src/stories/components/forms/textarea/textarea.snapshot.stories.ts

```diff
@@ -47,27 +47,9 @@ export const Textarea: Story = {
               ${getCombinations('size', context.argTypes.size.options, combinations).map(
                 (args: Args) => {
                   context.id = `${bg}-${crypto.randomUUID()}`;
-                  return html`
-                    <div>
-                      ${args.title !== undefined && args.title
-                        ? html`
-                            <h4>
-                              ${Object.entries(context.argTypes.size.control.labels)
-                                .filter(([key]) => key === args.size)
-                                .map(s => s[1])}
-                            </h4>
-                          `
-                        : ''}
-                      <div>${meta.render?.({ ...context.args, ...args }, context)}</div>
-                    </div>
-                  `;
+                  return html` <div>${meta.render?.({ ...context.args, ...args }, context)}</div> `;
```

Comment:
I think you removed too much here. All of the floating labels variants have disappeared.